.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateTimePeriodExtractor.cs

```diff
@@ -90,8 +90,10 @@ private IEnumerable<Token> MergeDateWithTimePeriodSuffix(string text, List<Extra
                     break;
                 }
 
-                if (ers[i].Type.Equals(Constants.SYS_DATETIME_DATE, StringComparison.Ordinal) &&
-                    ers[j].Type.Equals(Constants.SYS_DATETIME_TIME, StringComparison.Ordinal))
+                if ((ers[i].Type.Equals(Constants.SYS_DATETIME_DATE, StringComparison.Ordinal) &&
+                    ers[j].Type.Equals(Constants.SYS_DATETIME_TIME, StringComparison.Ordinal)) ||
+                    (this.config.CheckBothBeforeAfter && ers[j].Type.Equals(Constants.SYS_DATETIME_DATE, StringComparison.Ordinal) &&

```

Comment:
What does this specific change represent?

Reply:
In the original json file, certain cases, such as "2:00'dan sonra 1/1/2015'te olacak." (It will happen 1/1/2015 after 2:00), had the time entity preceding the date entity, so we included this possibility in the if statement. 
However later, the structure of these cases was updated ("1/1/2015'te 2:00'dan sonra olacak.") to have the date entity preceding the time entity since it is more common and meaningful. 
So, the above modification (and the corresponding one in BaseDateTimePeriodParser, line 432) are no longer necessary and we will remove them.